client/state/billing-transactions/schema.js

```diff
@@ -6,47 +6,12 @@ export const billingTransactionsSchema = {
 			type: 'array',
 			items: {
 				type: 'object',
-				properties: {
-					id: { type: 'string' },
-					service: { type: 'string' },
-					amount: { type: 'string' },
-					icon: { type: 'string' },
-					date: { type: 'string' },
-					desc: { type: 'string' },
-					org: { type: 'string' },
-					url: { type: 'string' },
-					support: { type: 'string' },
-					pay_ref: { type: 'string' },
-					pay_part: { type: 'string' },
-					cc_type: { type: 'string' },
-					cc_num: { type: 'string' },
-					cc_name: { type: 'string' },
-					cc_email: { type: 'string' },
-					credit: { type: 'string' },
-					items: {
-						type: 'array',
-						items: {
-							type: 'object'
-						}
-					}
-				},
 			},
 		},
 		upcoming: {
 			type: 'array',
 			items: {
 				type: 'object',
-				properties: {
-					id: { type: 'string' },
-					blog_id: { type: 'string' },
-					plan: { type: 'string' },
-					amount: { type: 'string' },
-					date: { type: 'string' },
-					product: { type: 'string' },
-					interval: { type: 'string' },
-					icon: { type: 'string' },
-					domain: { type: 'string' },
```

Comment:
From the other branch. It's upset because domain might be a string or null:
![screen shot 2017-01-18 at 10 55 24 am](https://cloud.githubusercontent.com/assets/1270189/22078341/898224fc-dd6c-11e6-9b75-7246177ce7c3.png)

If you update the schema to:
```
{ domain: { type: [ 'string', 'null' ] }
```

If you'd still like to persist data I'd recommend double checking this endpoint to see if any other mixed types are returned.